src/panels/config/logs/error-log-card.ts

```diff
@@ -74,24 +91,35 @@ class ErrorLogCard extends LitElement {
   private async _refreshErrorLog(): Promise<void> {
     this._errorHTML = this.hass.localize("ui.panel.config.logs.loading_log");
     const log = await fetchErrorLog(this.hass!);
+    this._isLogLoaded = true;
 
     this._errorHTML = log
-      ? log.split("\n").map((entry) => {
-          if (entry.includes("INFO"))
-            return html`<div class="info">${entry}</div>`;
-
-          if (entry.includes("WARNING"))
-            return html`<div class="warning">${entry}</div>`;
-
-          if (
-            entry.includes("ERROR") ||
-            entry.includes("FATAL") ||
-            entry.includes("CRITICAL")
+      ? log
+          .split("\n")
+          .filter(
+            memoizeOne((entry) => {
```

Comment:
No need to use `memoizeOne` here, as we only do this if we fetch new logs.

Reply:
Thanks, removed.